README.md

```diff
@@ -292,6 +292,61 @@ Some have opted to toggle the storybook component by using a custom option in th
 - [Heres an approach for react native cli](https://dev.to/dannyhw/multiple-entry-points-for-react-native-storybook-4dkp)
 - [Heres an article about how you can do it in expo](https://dev.to/dannyhw/how-to-swap-between-react-native-storybook-and-your-app-p3o)
 
+## withStorybook wrapper
+
+`withStorybook` is a wrapper function for your metro config. It accepts your existing [Metro config](https://metrobundler.dev/docs/configuration), and supports a variety of options for how Storybook should be started and configured.
+
+```js
+// metro.config.js
+const { getDefaultConfig } = require('expo/metro-config');
+const withStorybook = require('@storybook/react-native/metro/withStorybook');
+
+const defaultConfig = getDefaultConfig(__dirname);
+
+module.exports = withStorybook(defaultConfig, {
+  enabled: true,
+  // See API section below for available options
+});
+```
+
+### Options
+
+#### enabled
+
+Type: `boolean`, default: `true`
+
+Whether or not to show Storybook in the UI. Can by dynamically configured by a `process.env` variable.
```

Comment:
```suggestion
Configures the UI to display Storybook. It can be dynamically configured via `process.env` variable.
```

Reply:
hmm actually this is kinda both inaccurate, the option will actually just change if the metro options are applied or not, showing the ui or not is controlled by the user in the main entrypoint

Reply:
@dannyhw, are you ok with something of this nature?

```suggestion
Allows Storybook's configuration to be applied to the project by default without overriding the existing configuration when switching between environments.
```